stdlib/std.ncl

```diff
@@ -909,8 +909,8 @@
 
     label
       | doc m%"
-          The label submodule gathers functions that manipulate the label of
-          a contract.
+          The label submodule provides functions that manipulate the label of a
+          contract.
```

Comment:
```suggestion
          The label submodule provides functions that manipulate the label
          associated to a contract application.
```

Reply:
@vkleen do you think that'd be ok?

Reply:
I'd say "associated with" (because otherwise my thesis adviser will come haunt me) and "manipulates". Otherwise, looks good 👍